app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -39,47 +41,53 @@ def ems_inv_to_hashes
     private
 
     def get_ems
-      return if @inventory[:ems].nil?
-
-      @ems.api_version = normalize_blank_property(@inventory[:ems][:Props][:ServerInterfaceVersion])
-      @ems.uid_ems     = normalize_blank_property(@inventory[:ems][:Props][:ManagedComputer][0][:Props][:ID])
+      @ems.api_version = @inventory['ems']['Version']
+      @ems.uid_ems = @inventory['ems']['Guid']
     end
```

Comment:
@djberg96 
What will happen if there are no `@inventory[:ems]`, `Version` or `Guid` in the results?